components/list/list-item-mixin.js

```diff
@@ -460,6 +465,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 		if (changedProperties.has('breakpoints')) {
 			this.resizedCallback(this.offsetWidth);
 		}
+		if (changedProperties.has('_siblingHasColor') || changedProperties.has('color')) {
```

Comment:
I _think_ this computed reactive property will result in a subsequent iteration of the render life cycle. One way to avoid that would be to compute `_renderColorSlot` before `render` instead of after by using the `willChange` callback. 

```javascript
willUpdate(changedProperties) {
  // only need to check changed properties for expensive computation (according to Lit docs)
  this._renderColorSlot = this.color || this._siblingHasColor;
}
```

An alternative would be to sprout `_siblingHasColor` and update the CSS to check for both `_sibling-has-color` and `color`, thereby avoiding computing and sprouting `_renderColorSlot` altogether. Seems like `_renderColorSlot` really just buys us slightly simpler CSS.

Nit: on naming. If we keep `_renderColorSlot`, I'd probably rename it to something like `_hasColorSlot` or something along those lines, since anything that starts with `_render` sounds like a method that renders something and is how most of our private render helpers are named. I'd probably also rename `_siblingHasColor` to `_hasSiblingColor`. I think Ian introduced the XHasY naming here... I probably let it go because he had a lot of other more important feedback to address.

Reply:
Btw, I see Ian. introduced the `_render` as well. So what you have for naming is consistent with that. I'd almost be torn about being inconsistent with that. Anyway, main thing above is to be careful about the re-renders that computed properties can introduce.